agent/hibinit-agent

```diff
@@ -353,55 +302,53 @@ def identify_file_system(swapfile, file_system):
 
 
 class Config(object):
-    def __init__(self, config, args):
-        def get(section, name):
+    def __init__(self, config_file, args):
+        def get_from_config(section, name):
             try:
-                return config.get(section, name)
+                return config_file.get(section, name)
             except NoSectionError:
                 return None
             except NoOptionError:
                 return None
 
-        def get_int(section, name):
-            v = get(section, name)
-            if v is None:
-                return None
-            return int(v)
+        def get_int_from_config(section, name):
+            v = get_from_config(section, name)
+            return None if v is None else int(v)
 
         self.log_to_syslog = self.merge(
-            self.to_bool(get('core', 'log-to-syslog')), self.to_bool(args.log_to_syslog), True)
+            self.to_bool(args.log_to_syslog),
+            self.to_bool(get_from_config('core', 'log-to-syslog')),
+            True)
+        self.grub_update = self.merge(
+            self.to_bool(args.grub_update),
+            self.to_bool(get_from_config('core', 'grub-update')),
+            True)
 
-        self.mkswap = self.merge(get('swap', 'mkswap'), args.mkswap, 'mkswap {swapfile}')
-        self.swapon = self.merge(get('swap', 'swapon'), args.swapon, 'swapon {swapfile}')
-        self.swapoff = self.merge(get('swap', 'swapoff'), args.swapoff, 'swapoff {swapfile}')
         self.touch_swap = self.merge(
-            self.to_bool(get('core', 'touch-swap')), self.to_bool(args.touch_swap),
+            self.to_bool(args.touch_swap),
+            self.to_bool(get_from_config('core', 'touch-swap')),
             identify_file_system(SWAP_FILE, "xfs"))
```

Comment:
If SWAP_FILE is constant global variable why are we passing it  as an arguments ?

Reply:
Got it. I believe that is okay to do. But making such change earlier is something I generally try to avoid simply because if we cant make the future change it leaves us in an undesirable state. 

Having said that, I also think this is not a big issue and I will be fine with this